### PR TITLE
Reader: add a consistent indent for subscription list avatars

### PR DESCRIPTION
**Pull request by bluefuton**
If a subscription list item is missing an avatar, indent it anyway. This makes scanning down the list much easier and potentially fixes https://github.com/Automattic/wp-calypso/issues/14083.

Before:

screen shot 2017-05-19 at 16 04 14

After:

screen shot 2017-05-19 at 16 03 46

### CODE REVIEW

File: client/blocks/reader-subscription-list-item/style.scss [hunk 1]
```diff
@@ -305,3 +305,8 @@
 		}
 	}
 }
+
+.reader-subscription-list-item__avatar {
```
Comment (blowery) [inline client/blocks/reader-subscription-list-item/style.scss]: clever! i like it.

File: client/reader/following-manage/feed-search-results.jsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ const FollowingManageSearchFeedsResults = ( {
 	searchResultsCount,
 	query,
 } ) => {
-	const isEmpty = !! ( query && searchResults && searchResults.length === 0 );
+	const isEmpty = !! ( query && query.length > 0 && searchResults && searchResults.length === 0 );
```
Comment (blowery) [inline client/reader/following-manage/feed-search-results.jsx]: unrelated bug fix?

Comment (bluefuton) [inline client/reader/following-manage/feed-search-results.jsx]: Yep - shall I add to another PR? Should stop a flash of 'no sites matched %s' when the page is loading.

Comment (blowery) [inline client/reader/following-manage/feed-search-results.jsx]: no, s'fine. just wanted to make sure i understood.

Comment (samouri) [inline client/reader/following-manage/feed-search-results.jsx]: sneaky sneaky
